app/views/events/_form.html.erb

```diff
@@ -26,10 +26,23 @@
     <%= form.datetime_local_field :started_at %>
   </div>
 
+<!--
   <div class="field">
     <%= form.label :place %>
     <%= form.select :place_id, options_from_collection_for_select(Place.all, :id, :full_adress)%>
   </div>
+  -->
+  <%= form.fields_for :place_attibutes, event.place do |place_form| %>
```

Comment:
It is funny, because this code is almost OK. The only thing is missing here is one `r` letter. We should have here `place_attributes,`